lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -514,6 +514,13 @@ function dosomething_global_get_language($account, stdClass $node = NULL, $respe
     // Respect the URL first.
     $request_path = explode('/', request_path());
     if (!empty($request_path[0])) {
+
+      // If the string is longer than a country code
```

Comment:
`campaigns` -- but if the default language was Spanish/Mexico, this would be `voluntario`